virtio-devices/src/vhost_user/vu_common_ctrl.rs

```diff
@@ -36,7 +37,7 @@ use crate::{
 // Size of a dirty page for vhost-user.
 const VHOST_LOG_PAGE: u64 = 0x1000;
 
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, PartialEq, Eq, serde::Serialize, Deserialize)]
```

Comment:
nit: Is there are reason why you can't use `serde::Serialize` and then use the short form here?